framework/wazuh/core/tests/test_rule.py

```diff
@@ -29,7 +29,7 @@
 
 ruleset_conf = {
     'decoder_dir': ['ruleset/decoders', 'etc/decoders'],
-    'rule_dir': ['ruleset/rules', 'etc/rules'], 'rule_exclude': ['0215-policy_rules.xml'],
+    'rule_dir': ['framework/wazuh/core/tests/data/rules'], 'rule_exclude': ['0215-policy_rules.xml'],
```

Comment:
Is this path, and the format of the array correct?
**decoder_dir**, i.e., uses two elements for the array and target to the folders where the rules/decoders are originally placed.

Reply:
All ok, the test passed correctly